render/tests/test-updateElement.js

```diff
@@ -179,19 +179,6 @@ o.spec("updateElement", function() {
 		o(updated.dom.style.backgroundColor).equals("red")
 		o(updated.dom.style.border).equals("")
 	})
-	o("updates style when it's same object but mutated", function() {
```

Comment:
Could you reinstate this test and just test for the opposite (that it *doesn't* update)?

Reply:
I'm not sure it's a good idea to write tests to validate undefined behaviour. Not respecting mutations is something we warn against at a high level, but it's not something we should be asserting enforcements for at every level.